docs/content/integrations/powerbi.mdx

```diff
@@ -30,19 +30,20 @@ This guide provides instructions for using Dagster with Power BI. Your Power BI
 
 To load Power BI assets into the Dagster asset graph, you must first construct a <PyObject module="dagster_powerbi" object="PowerBIWorkspace" /> resource, which allows Dagster to communicate with your Power BI workspace. You'll need to supply your workspace ID and credentials. You may configure a service principal or use an API access token, which can be passed directly or accessed from the environment using <PyObject object="EnvVar" />.
 
-Dagster can automatically load all semantic models, data sources, reports, and dashboards from your Power BI workspace. Call the <PyObject module="dagster_powerbi" object="PowerBIWorkspace" method="build_defs" /> function, which returns a <PyObject object="Definitions" /> object containing all the asset definitions for these Power BI assets.
+Dagster can automatically load all semantic models, data sources, reports, and dashboards from your Power BI workspace as asset specs. Call the <PyObject module="dagster_powerbi" method="load_powerbi_asset_specs" /> function, which returns a list of <PyObject object="AssetSpec" />s representing your Power BI assets. You can then include these asset specs in your <PyObject object="Definitions" /> object:
```

Comment:
`load_powerbi_asset_specs` link does not work; if this is new make sure to run `make apidoc-build`.

<img width="315" alt="image" src="https://github.com/user-attachments/assets/5d5db40d-4bf5-4b4c-938e-dee6f674af0e">